e2e/wdio/platform/tests/date-picker.e2e-spec.ts

```diff
@@ -0,0 +1,243 @@
+import {
+    click,
+    elementArray, getAttributeByName, getCSSPropertyByName, getText, getValue,
+    refreshPage, scrollIntoView, sendKeys, setValue,
+    waitForElDisplayed, waitForPresent
+} from '../../driver/wdio';
+import { DatePicker } from '../pages/date-picker.po';
+import datePickerTestData from '../fixtures/testData/date-picker';
+
+describe('Date picker suite', function() {
+    const datePickerPage: DatePicker = new DatePicker();
+
+    beforeAll(() => {
+        datePickerPage.open();
+        waitForPresent(datePickerPage.inputDatePicker);
```

Comment:
I think you could move this waitForPresent to the open method in the PO file; I did it like that for all other test specs.